src/Concretize.hs

```diff
@@ -1388,7 +1388,7 @@ manageMemory typeEnv globalEnv root =
         else pure (Right xobj)
     unmanageArg xobj@XObj {} =
       pure (Right xobj)
-    createDeleter :: XObj -> Maybe Deleter
+    getDropFunc t = nameOfPolymorphicFunction typeEnv globalEnv (FuncTy [(RefTy t (VarTy "w"))] UnitTy StaticLifetimeTy) "drop" -- TODO: var is not sound
```

Comment:
You can create a lifetime from the xobj here by using `makeTypeVariableNameFromInfo`, it will be ugly but unique.

Reply:
Thanks for that hint, I didn’t know that! Fixed.